examples/multi-datacenter/submit_replicator_dc1_to_dc2_topic2.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+HEADER="Content-Type: application/json"
+DATA=$( cat << EOF
+{
+  "name": "replicator-dc1-to-dc2-topic2",
+  "config": {
+    "connector.class": "io.confluent.connect.replicator.ReplicatorSourceConnector",
+    "topic.whitelist": "topic2",
+    "topic.rename.format": "\${topic}.replica",
+    "key.converter": "io.confluent.connect.replicator.util.ByteArrayConverter",
+    "value.converter": "io.confluent.connect.replicator.util.ByteArrayConverter",
```

Comment:
key.converter, value.converter and header.converter are all worker level config. They should be set in docker-compose.yml, not in the connector configuration.